spec/acceptance/cli/install_spec.rb

```diff
@@ -52,19 +52,24 @@
     it 'accepts --jobs option to set job size' do
       output = run 'appraisal install --jobs=2'
 
-      expect(output).to include
-        'bundle install --gemfile=gemfiles/1.0.0.gemfile --jobs=2'
+      expect(output).to include(
+        "bundle install --gemfile='#{file('gemfiles/1.0.0.gemfile')}' --jobs=2"
+      )
     end
 
     it 'ignores --jobs option if the job size is less than or equal to 1' do
       output = run 'appraisal install --jobs=0'
 
+      expect(output).
+        to include(
+        "bundle install --gemfile='#{file('gemfiles/1.0.0.gemfile')}'"
+      )
       expect(output).not_to include(
-        'bundle install --gemfile=gemfiles/1.0.0.gemfile')
+        "bundle install --gemfile='#{file('gemfiles/1.0.0.gemfile')}' --jobs=0"
+      )
       expect(output).not_to include(
-        'bundle install --gemfile=gemfiles/1.0.0.gemfile --jobs=0')
-      expect(output).not_to include(
-        'bundle install --gemfile=gemfiles/1.0.0.gemfile --jobs=1')
+        "bundle install --gemfile='#{file('gemfiles/1.0.0.gemfile')}' --jobs=1"
```

Comment:
I see Hound is not enabled, so this line (and similar ones above) is not being barked at for not ending on a comma. Is that ok by you?

Reply:
Unfortunately, that's an error under Ruby 1.8.7, which is still being run in CI. Removing the trailing comma fixes this.

One option here would be to first go about dropping support for 1.8.7. What do you think?

Reply:
Hm. One one hand, I'm for dropping 1.8.7, but on the other hand I don't find coding style a good enough reason for it.

I'm happy to leave it as is for now.